### PR TITLE
Add heading element

### PR DESCRIPTION
**Pull request by dsager**
A heading is created from a `h1`, `h2`, `h3`, `h4` or `h5` tag.

### CODE REVIEW

**Comment by christofdamian** [comment]
+1

File: lib/article_json/import/google_doc/html/heading_element.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          # Determine the level of the heading
+          # The level corresponds to the header tag, e.g. `<h3>` is level 3.
+          # @return [Integer]
+          def level
+            case @node.name
+              when 'h1' then 1
+              when 'h2' then 2
+              when 'h3' then 3
+              when 'h4' then 4
+              when 'h5' then 5
```
Comment (towanda) [inline lib/article_json/import/google_doc/html/heading_element.rb:26]: What about:

```ruby
@node.name.tr('h','').to_i
```

Comment (dsager) [reply lib/article_json/import/google_doc/html/heading_element.rb:26]: then `h42` would also work :)

Comment (towanda) [reply lib/article_json/import/google_doc/html/heading_element.rb:26]: and that would be a feature, right?

**Comment by towanda** [comment]
+!

**Comment by towanda** [comment]
+1

**Pull request merged**